communis-services/src/main/java/com/rhema/communis/CommunisServicesApplication.java

```diff
@@ -2,8 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.scheduling.annotation.EnableScheduling;
 
 @SpringBootApplication
+@EnableScheduling
```

Comment:
Nice, what are we going to queue?

Reply:
This is from pulling from upstream master. @robinvk6 would know